### DISCUSSION TITLE
FluentDataGrid - tooltiptext set background colour

### DISCUSSION THREAD

**Discussion by RobDene** [microsoft/fluentui-blazor#2647] [General]
Can anyone let me know how to set this? We are scratching our heads 😁

**Comment by dvoituron** [comment]
Are you talking about this tooltip?

![{3C5658C5-1F79-4FC2-8244-369F88A88B01}](https://github.com/user-attachments/assets/9adbca64-de7f-44ea-b9b8-4a081f7e9c88)

For this Tooltip, we are using the HTML `title` attribute. Ufortunately, you can’t directly customize the background color of a tooltip that appears when using the title attribute, as this tooltip is rendered by the browser and its appearance is controlled by the browser’s default styles.

**Reply by RobDene** [reply]
Yes , thats the one - we coded to provide dynamic data but can't work out background

**Reply by RobDene** [reply]
Can I request  as you can change the other hovers and tooltips  that this one be included ?

**Comment by dvoituron** [comment]
I think the only possibility will be to customize your headers using `HeaderCellItemTemplate`.
@vnbaaij any other ideas?

**Reply by vnbaaij** [reply]
Currently, yes that is the only option

I'm looking at some grid work so will look at changing this to use our own tooltip for this
